pkgs/tools/misc/viddy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+  pname = "viddy";
+  owner = "sachaos";
+  version = "0.2.1";
+in
+buildGoModule {
+  name = pname;
+  inherit version;
```

Comment:
```suggestion
  inherit version pname;
```

Reply:
Thanks for the suggestion! I changed the let-binding to just `name`, and am now inheriting both directly

Reply:
no, we should use `pname` instead of just `name`

Reply:
fixed, is this ok?

Reply:
Looks good to me.